app/views/carts/show.html.erb

```diff
@@ -1,17 +1,6 @@
 <div class="card">
   <h4 class="card-header text-center">Your Cart</h4>
-  <div class="card-body">
-    <ul class= "list-group list-group-flush">
-      <% items.each do |item| %>
-        <li class="list-group-item">
-          <h4>
-            <%= item.product.name %>
-          </h4>
-          <%= number_to_currency item.product.price %>
-        </li>
-      <% end %>
-    </ul>
-  </div>
+  <%= render partial: 'carts/items_list', locals: { items: items, total_presenter: cart_presenter } %>
```

Comment:
```suggestion
  <%= render partial: 'carts/items_list', locals: { items: items, total_presenter: cart_presenter.total } %>
```

Reply:
and change it in the partial?

Reply:
What about 
```suggestion
  <%= render partial: 'carts/items_list', locals: { items: items, total_items_price: cart_presenter.total_price } %>
```
to make it clearer, that it is only the total price and not the whole presenter?

Reply:
You're right, good catch.
I was confused because you set `total_presenter: cart_presenter` -> it was still a cart_presenter, but currently I think if you need only total_price, your changes looks good